src/util.js

```diff
@@ -211,6 +211,13 @@ export async function getSchedule(xo, id) {
   return schedule;
 }
 
+export async function getScheduleId(params) {
+  const backupNgJob = await xo.call("backupNg.getJob", params);
```

Comment:
Why not simply use `schedule.getAll`?

Reply:
You're right. I didn't pay attention :+1: